fe/fe-core/src/main/java/com/starrocks/http/meta/GlobalDictMetaService.java

```diff
@@ -91,16 +91,26 @@ public void executeInLeaderWithAdmin(BaseRequest request, BaseResponse response)
                     writeResponse(request, response, HttpResponseStatus.BAD_REQUEST);
                     return;
                 }
-
-                boolean isEnable = "true".equalsIgnoreCase(request.getSingleParameter(ENABLE).trim());
-
+                boolean isEnable = true;
+                String enableParamValue = request.getSingleParameter(ENABLE);
+                if (!Strings.isNullOrEmpty(enableParamValue)) {
```

Comment:
This api explicitly enables or disables the `{db}.{table}`'s global dict. reading from the original code, the `enable` parameter should be required. It is better implemented that if the `enable` parameter is not given, `HTTP 400 BAD Request` is responded, the same way as the `db_name`, `table_name` check.

Reply:
Hello @kevincai Many thanks, I tried to address your suggestions.